packages/loopring_v3/contracts/amm/LoopringAmmPool.sol

```diff
@@ -212,8 +212,10 @@ contract LoopringAmmPool is
         )
         external
         nonReentrant
+        onlyWhenOnline
         onlyFromAssetManager
     {
+        require(!state.exitMode, "DISALLOWED_IN_EXIT_MODE");
```

Comment:
do we need these changes？

Reply:
I don't think we can do these checks because the pool may be offline while the asset manager still needs to return funds to the pool and so `balancesL1` still needs updating. Same for when the pool is in exit mode, funds still may need to be returned to the pool.

Reply:
OK